packages/core/src/components/dyte-participants-stage-queue/dyte-participants-stage-queue.tsx

```diff
@@ -78,8 +78,7 @@ export class DyteParticipantsStaged {
   private shouldShowStageRequests = () => {
     return (
       this.meeting.self.permissions.stageEnabled &&
-      this.meeting.self.permissions.acceptStageRequests &&
-      this.stageRequestedParticipants.length > 0
```

Comment:
do we want to show the requests tab when there are no requests?

Reply:
Not showing should be fine

Reply:
Done.